bool accept
uint32 code
---
bool success
string message
